commands/commands_test.go

```diff
@@ -35,7 +35,7 @@ import (
 
 // Redirecting stdOut so we can analyze output line by
 // line and check with what we want.
-var stdOut *os.File = os.Stdout
+var stdOut = os.Stdout // *os.File
 
```

Comment:
Remove also the comment `// *os.File`

Reply:
Done 3b2517935dda8b02d5eb42c7e42100b0841c89db